### DISCUSSION TITLE
Glitch on git bash after using for a while.

### DISCUSSION THREAD

**Discussion by RiddlerKnight** [JanDeDobbeleer/oh-my-posh#4245] [Troubleshoot] [answered]
![2023-09-09_14-14-31](https://github.com/JanDeDobbeleer/oh-my-posh/assets/15905772/54496028-ca55-4eca-8cc3-54bd7d0d1aae)

I'm not sure that I config it correctly. I try search this issue but found nothing.

**Comment by JanDeDobbeleer** [comment, marked as answer]
the glitch is terminal resize, that never re-renders the prompt. That said, you have a right and left prompt without a newline so you'll always start typing after the last rendered block.
